packages/vuetify/src/labs/VNumberInput/VNumberInput.tsx

```diff
@@ -121,12 +121,35 @@ export const VNumberInput = genericComponent<VNumberInputSlots>()({
 
     const decrementSlotProps = computed(() => ({ click: onClickDown }))
 
+    // Control holding down states
+    let hldDwnReqNextTime = 0
+    let hldDwnReq = -1
+    const hldDwnReqDelay = 100
+    const hldDwnDelay = 500
+    let hldDwn: null | 'up' | 'down' = null
+    let hldDwnTimeout = -1
+
+    function controlHoldingDownWatcher (time: number) {
+      hldDwnReq = requestAnimationFrame(controlHoldingDownWatcher)
+      if (time < hldDwnReqNextTime) return
+      hldDwnReqNextTime = time + hldDwnReqDelay
+
+      if (hldDwn) {
+        toggleUpDown(hldDwn === 'up')
+      }
+    }
+
     onMounted(() => {
+      hldDwnReq = requestAnimationFrame(controlHoldingDownWatcher)
```

Comment:
Why is this running constantly instead of only while the button is held?

Reply:
Thanks, refactored